### PR TITLE
FUSETOOLS-2267 - Code cleaning

### PR DESCRIPTION
**Pull request by apupier**
targeting version after 9.1.0

### CODE REVIEW

**Comment by apupier** [comment]
I need to rework on this PR before review as I worked on it to unblock release situation

File: jmx/plugins/org.fusesource.ide.jmx.karaf/src/org/fusesource/ide/jmx/karaf/connection/ui/AutodetectedKarafLabelProvider.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2014 Red Hat, Inc.
+  * Copyright (c) 2014 Red Hat, Inc.
```
Comment (lhein) [inline jmx/plugins/org.fusesource.ide.jmx.karaf/src/org/fusesource/ide/jmx/karaf/connection/ui/AutodetectedKarafLabelProvider.java]: is that an intentional change?

File: jmx/plugins/org.fusesource.ide.jmx.karaf/src/org/fusesource/ide/jmx/karaf/navigator/osgi/BundlesTableSheetPage.java [hunk 2]
```diff
@@ -246,7 +246,7 @@ protected void setSelectedBundleIds(Set<Long> selectedBundleIds) {
 		TableViewer viewer = getTableView().getViewer();
 		if (viewer != null) {
 			List<?> propertySources = getPropertySources();
-			List selected = new ArrayList();
+			List<Object> selected = new ArrayList<Object>();
```
Comment (lhein) [inline jmx/plugins/org.fusesource.ide.jmx.karaf/src/org/fusesource/ide/jmx/karaf/navigator/osgi/BundlesTableSheetPage.java]: List< Object > selected = new ArrayList<>();  ?

File: jmx/plugins/org.fusesource.ide.jmx.karaf/src/org/fusesource/ide/jmx/karaf/navigator/osgi/ServicesTableSheetPage.java [hunk 3]
```diff
@@ -47,7 +46,7 @@ public void setView(IViewPage view) {
 	@Override
 	public void setActionBars(IActionBars actionBars) {
 		super.setActionBars(actionBars);
-		IMenuManager menu = actionBars.getMenuManager();
+		//IMenuManager menu = actionBars.getMenuManager();
```
Comment (lhein) [inline jmx/plugins/org.fusesource.ide.jmx.karaf/src/org/fusesource/ide/jmx/karaf/navigator/osgi/ServicesTableSheetPage.java]: why keeping it? Code smell?

Comment (apupier) [inline jmx/plugins/org.fusesource.ide.jmx.karaf/src/org/fusesource/ide/jmx/karaf/navigator/osgi/ServicesTableSheetPage.java]: Effectively, bad code smell ^^
actionBars.getMenuManager() is also initilizating the field so I will keep the call but just remove the assignation

**Pull request merged**